explainer.md

```diff
@@ -64,7 +68,7 @@ In addition to the `Struct` type [described below](#struct-type-definitions), Ty
     uint32 int32 float32 object
     uint64 int64 float64
```

Comment:
How about removing the intro sentence and say instead: "The following common value type definitions are provided by the JS spec" and then include `ref` in the list, explaining it later.